service/db/src/test/java/gov/va/vro/service/db/SaveToDbServiceImplTest.java

```diff
@@ -40,71 +35,146 @@ class SaveToDbServiceImplTest {
 
   @Autowired private ClaimRepository claimRepository;
 
+  @Autowired private ClaimSubmissionRepository claimSubmissionRepository;
+
   @Autowired private AssessmentResultRepository assessmentResultRepository;
 
+  @Autowired private ExamOrderRepository examOrderRepository;
+
   @Value("classpath:test-data/evidence-summary-document-data.json")
   private Resource esdData;
 
   @Test
-  void persistClaim() {
+  void persistClaimWithV1Data() {
     Claim claim = new Claim();
-    claim.setClaimSubmissionId("claim1");
+    claim.setCollectionId("collection1");
+    claim.setIdType(Claim.V1_ID_TYPE);
     claim.setVeteranIcn("v1");
     claim.setDiagnosticCode("1234");
     var result = saveToDbService.insertClaim(claim);
     assertNotNull(result.getRecordId());
-    assertEquals(claim.getClaimSubmissionId(), result.getClaimSubmissionId());
+    assertEquals(claim.getBenefitClaimId(), result.getBenefitClaimId());
```

Comment:
I was intentionally ensuring that the values were null. That said, result no longer *has* the claimSubmissionId on it in this line. it was moved to the other table so that line *must* be removed or changed. I think further rename refinements really need to wait to a later PR. 

I have added a comment however that this *really is* ensuring that null is a thing.

Reply:
The subsequent line where we check claimSubmissionRepository for the collectionId or throw, is testing the equivalent of this line that we set the correct value on the correct table.